docs/source/markdown/podman-volume-ls.1.md

```diff
@@ -18,13 +18,15 @@ flag. Use the **--quiet** flag to print only the volume names.
 
 Volumes can be filtered by the following attributes:
 
-- dangling
-- driver
-- label
-- name
-- opt
-- scope
-- until
+| **Filter** | **Description**                                                                       |
+| ---------- | ------------------------------------------------------------------------------------- |
+| dangling   | [Dangling] Matches all volumes not referenced by any containers                       |
+| driver     | [Driver] Matches volumes based on their driver                                        |
+| label      | [Key] or [Key=Value] Label assigned to a volume                                       |
+| name       | [Name] Volume name (accepts regex)                                                    |
+| opt        | Matches a storage driver options                                                      |
+| scope      | Filters volume by scope                                                               |
+| until      | Only remove networks created before given timestamp                                   |
```

Comment:
Is this correct? I think "Only match volumes created before given timestamp" would be more correct

Reply:
Yes... I wrote network and should be volumes. I will correct.